### DISCUSSION TITLE
[Start here] Checkout branding feedback

### DISCUSSION THREAD

**Discussion by akatigbak** [Shopify/checkout-branding#1] [Announcements]
The Checkout team is looking for your feedback on all things checkout branding! Here are some questions we’d like to get your thoughts on:

- What was your general experience using the branding API? Did you find anything particularly challenging or delightful?
- What would you like to see as additional style controls to checkout branding? 
- Is there any other feedback you want to share? What can we add, remove, or change to help you customize checkout's look and feel?

**Comment by kylebuildsstuff** [comment]
Centering a logo using the branding logo alignment setting doesn't seem to work properly. 

**Edit**: Moved details to issue for easier tracking https://github.com/Shopify/checkout-branding/issues/3

**Comment by devinenoise** [comment]
I'm struggling to find a mutation in the CheckoutBrandingCustomizationsInput that will set the Discount APPLY button to the same height as the input field next to it. I have tried primaryButton blockPadding all the way to EXTRA_TIGHT and it still doesn't collapse enough to the same size as the input field.

**Reply by akatigbak** [reply]
hi there! It could be an issue on our end. Are you able to share a screenshot of what you're seeing so we can further investigate?

**Reply by devinenoise** [reply]
I was able to figure out that when setting the CheckoutBrandingLabelPosition OUTSIDE that it adds that much more height to the parent container, making the APPLY discount button also taller.  The fix for now was to set CheckoutBrandingLabelPosition to INSIDE.  Perhaps the apply discount button could be moved to a different container in the markup and then the CheckoutBrandingLabelPosition Customization wouldn't affect that button.
